executor/src/constant_evaluator/mod.rs

```diff
@@ -18,6 +20,9 @@ use rayon::prelude::{IntoParallelIterator, ParallelIterator};
 
 mod data_structures;
 
+pub const MIN_DEGREE_LOG: usize = 5;
+pub const MAX_DEGREE_LOG: usize = 10;
```

Comment:
These numbers are pretty arbitrary. I kept the maximum small for now, because the witness generation currently picks the largest size. Once that is fixed, the cost of increasing this number will be much lower.

Reply:
But what happens with the riscv tests if the max degree is 10 here?

Reply:
Currently, the ASM linker still sets the degree of all machines, so this is irrelevant for any ASM program.

I think, next, I'll do the witgen, so that it doesn't chose the largest anymore. Then, we can change this number.

After that, we can change the ASM linker and test ASM programs, including RISC-V.